### DISCUSSION TITLE
Dropped the project?

### DISCUSSION THREAD

**Discussion by SavinDevelop** [MasoniteFramework/masonite#758] [Q&A]
I am a young developer who is interested in mesonite and wanted to know if it is dead? Because there were no updates for a long time.

**Comment by yilanboy** [comment]
The author is taking a break because he's sick, you can check out his [twitter](https://twitter.com/JoeMancusoDev/status/1698346510830727368?t=kfZOnZLZNw5i3JOb_uYZxQ&s=19) (now X).

Wish for he a full recovery. 🥲

**Reply by SavinDevelop** [reply]
Oh, well i wish for he a full recovery

**Comment by josephmancuso** [comment]
i'm atill alive and will be upgrading the project now

**Reply by SavinDevelop** [reply]
This is thunder in the middle of a clear sky😍
I wish you success and strength
